### DISCUSSION TITLE
CSK_anomalyDetection

### DISCUSSION THREAD

**Discussion by TomerErezSick** [SICKAppSpaceCodingStarterKit/.github#5] [Ideas]
hey, a deep learning and multi deep learning csk modules were already published, but i think an anomalyDetection and multi anomalyDetection would be just as useful if not more. 
maybe also with the possibility to train the AD instances on device

**Comment by golluroSICKAG** [comment]
Hi Tomer,

thx for sharing your idea!
Do you feel ready to start to develop this feature by your own or are you looking for somebody to create this feature.

Kind regards:
Roy

**Reply by TomerErezSick** [reply]
i am currently adjusting one on my own, i can share it with you when i am done.
but i was wondering if someone (maybe on your team) plans to develop it.

**Reply by golluroSICKAG** [reply]
Hi Tomer,
was there any progress on this topic?
Kind regards:
Roy

**Reply by TomerErezSick** [reply]
Hey Roy, i have made an app for that but not in the csk standard and format unfortunately.
had i developed it a little later in time, i would have done it according to the csk format as i now understand the importance of these generic modules for app development.
i could share the work i have done with you if you want me to.
